spec/features/viewing_a_document_with_no_creator_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+RSpec.feature "Viewing a document with no creator" do
+  scenario "User finds document without a creator and views it" do
+    given_there_is_a_document_with_no_creator
+    when_i_visit_the_index_page
+    then_i_see_we_dont_know_who_created_the_document
+    when_i_visit_the_document_page
+    then_i_again_see_we_dont_know_who_created_the_document
+  end
+
+  def given_there_is_a_document_with_no_creator
+    @document = create(:document, title: "Excellent document", creator: nil)
```

Comment:
Is this title important here?

Reply:
It is not, thanks - I've simplified in c82f825